### PR TITLE
Fix for accelerometer simulator not working in firefox.

### PR DESCRIPTION
**Pull request by microsoftsam**
Fixes Microsoft/pxt#679

### CODE REVIEW

**Comment by pelikhan** [comment]
Will you propagate that fix to the other targets?

File: sim/visuals/microbit.ts [hunk 1]
```diff
@@ -712,8 +712,9 @@ namespace pxsim.visuals {
                     tiltDecayer = 0;
                 }
 
-                let ax = (ev.clientX - this.element.clientWidth / 2) / (this.element.clientWidth / 3);
-                let ay = (ev.clientY - this.element.clientHeight / 2) / (this.element.clientHeight / 3);
+                let bbox = this.element.getBoundingClientRect();
+                let ax = (ev.clientX - bbox.width / 2) / ( bbox.width / 3);
```
Comment (riknoll) [inline sim/visuals/microbit.ts]: whitespace

Comment (microsoftsam) [inline sim/visuals/microbit.ts]: Where?

Comment (riknoll) [inline sim/visuals/microbit.ts]: There is an extra space before the second `bbox.width`

Comment (microsoftsam) [inline sim/visuals/microbit.ts]: Oh I see it! Thanks

**Pull request merged**